packages/medical-domain/src/domains/repositories/medical/datum/cbg.service.ts

```diff
@@ -79,23 +80,31 @@ const normalizeBg = (rawData: Record<string, unknown>, opts: MedicalDataOptions,
     type: bgType,
     units: bgUnit,
     value: bgValue,
-    // only used for trends view ?
     ...getTrendsTime(base.epoch, base.timezone)
   }
   return bg
 }
 
 const normalize = (rawData: Record<string, unknown>, opts: MedicalDataOptions): Cbg => {
-  return normalizeBg(rawData, opts, DatumType.Cbg) as Cbg
+  return {
+    ...normalizeBg(rawData, opts, DatumType.Cbg),
+    type: DatumType.Cbg,
+    deviceName: rawData.deviceName ? rawData.deviceName as string : 'Unknown'
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❔ question:** ‏Should `Unknown` be translated?

Reply:
For now, this value is just used to determine what is the duration of one Cbg... So no need for translation.